tasks/build_app.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+- stat: path="{{ lookup('env','PWD') }}/docker-compose-build.yml"
+  register: docker_compose_build
+
+- name: "Build app's code with docker-compose if docker-compose-build.yml exists"
+  shell: "docker-compose -f docker-compose-build.yml up"
```

Comment:
In this step, just build the app is needed, right? why not using `docker-compose build` instead?